src/components/dashboard/ParticipantIndividual.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from "react";
+import { Box, Flex, SimpleGrid, Heading, Text, Accordion, AccordionItem, AccordionButton, AccordionPanel, AccordionIcon, Tag, TagLabel, TagLeftIcon, TagRightIcon, TagCloseButton } from "@chakra-ui/react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+const ParticipantIndividual: React.FC = () => {
+  const { applicationId } = useParams();
+  const [name, setName] = useState("name");
+  const [gmail, setGmail] = useState("gmail");
+  const [phone, setPhone] = useState("phone");
+  const [linkedin, setLinkedin] = useState("linkedin");
+  const [adult, setAdult] = useState(true);
+  const [school, setSchool] = useState("school");
+  const [major, setMajor] = useState("major");
+  const [graduation, setGraduation] = useState(0);
+  const [gender, setGender] = useState("gender");
+  const [ethnicity, setEthnicity] = useState("ethnicity");
+  const [applied, setApplied] = useState(false);
+  const [confirmed, setConfirmed] = useState(false);
+
+  useEffect(() => {
+    const getUserData = async () => {
+      try {
+        const app = await axios.get(
+          `https://registration.api.hexlabs.org/application/${applicationId}`
+        );
+        const user = await axios.get(
+          `https://users.api.hexlabs.org/users/'${app.data.applicationData.userId}`
+        )
+        setName(`${user.data.name.first}{' '}${user.data.name.last}`);
+        setGmail(user.data.email);
+        setPhone(app.data.applicationData.phoneNumber);
```

Comment:
For this, can you use `axios-hooks` instead? If you search the other files, you'll see examples of how we use the `useAxios` hook to retrieve data, and it simplifies a lot of things so you won't need to use state for each field.